site/content/api/Device-Connection-API.md

```diff
@@ -0,0 +1,148 @@
+
+A response to a *get last known gateway for device* request contains the [Standard Response Properties]({{< relref "#standard-response-properties" >}}).
+
+The response message includes payload with a JSON object having the following properties:
+
+| Name             | Mandatory | JSON Type     | Description |
+| :--------------- | :-------: | :------------ | :---------- |
+| *device-id*      | *yes*     | *string*      | The ID of the last known gateway for the device, or the device ID itself if the device doesn't support usage via a gateway. |
+| *update-date*    | *no*      | *string*      | The date that the information about the last known gateway for the device was last updated. The date is formatted in ISO-8601 format, e.g. '2011-12-03T10:15:30Z'. |
+
+The response message's *status* property may contain the following codes:
+
+| Code | Description |
+| :--- | :---------- |
+| *200* | OK, the payload contains the gateway or device ID. |
+| *404* | Not Found, there is no device registered with the given *device_id* within the given *tenant_id* or the device supports usage via a gateway but there was no last known gateway assigned to the device. |
+
+For status codes indicating an error (codes in the `400 - 499` range) the message body MAY contain a detailed description of the error that occurred.
+
+# Standard Message Properties
+
+Due to the nature of the request/response message pattern of the operations of the Device State API, there are some standard properties shared by all of the request and response messages exchanged as part of the operations.
+
+## Standard Request Properties
+
+The following table provides an overview of the properties shared by all request messages regardless of the particular operation being invoked.
+
+| Name             | Mandatory | Location                 | AMQP Type    | Description |
+| :--------------- | :-------: | :----------------------- | :----------- | :---------- |
+| *subject*        | yes       | *properties*             | *string*     | MUST be set to the value defined by the particular operation being invoked. |
+| *correlation-id* | no        | *properties*             | *message-id* | MAY contain an ID used to correlate a response message to the original request. If set, it is used as the *correlation-id* property in the response, otherwise the value of the *message-id* property is used. |
+| *message-id*     | yes       | *properties*             | *string*     | MUST contain an identifier that uniquely identifies the message at the sender side. |
+| *reply-to*       | yes       | *properties*             | *string*     | MUST contain the source address that the client wants to received response messages from. This address MUST be the same as the source address used for establishing the client's receive link (see [Preconditions]({{< relref "#preconditions" >}})). |
+| *content-type*   | yes       | *properties*             | *string*     | MUST be set to `application/json`. |
```

Comment:
we should remove this because
1. the request messages do not contain any payload anyway and
1. this limits compatibility with JMS based clients as there is no way to set the content type in JMS